must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-gather_common_ceph_resources "${BASE_COLLECTION_PATH}" "${SINCE_TIME}"
+gather_common_ceph_resources "$1" "$2"
 
 # Expect base collection path as an argument
 BASE_COLLECTION_PATH=$1
```

Comment:
It woud be better to first collect Path and time like:
BASE_COLLECTION_PATH=$1
SINCE_TIME=$2
Then declare common resources:  gather_common_ceph_resources "${BASE_COLLECTION_PATH}" "${SINCE_TIME}"